resources/dicts/events/death/mountainous.json

```diff
@@ -27,7 +27,7 @@
         "sub_type": [],
         "tags": ["clan:kitten"],
         "weight": 20,
-        "event_text": "m_c sneaks out of camp but finds {PRONOUN/m_c/self} lost in the strange, angular tunnels that lead out into the territory. The Clan searches for {PRONOUN/m_c/object} in the morning to no avail. The leftover scent of blood and mountain weasel serves as an intense warning to the other kits.",
+        "event_text": "m_c snuck out of camp and was lost to the strange tunnels, nothing left to be found but blood and weasel scent.",
```

Comment:
a small nitpick, but "the strange tunnels" makes it sound as though it's a known location though it hasn't been mentioned before. maybe something like "the winding tunnels" instead?

Reply:
hmm I don't mind changing it; but for added context this is a Ruins specific event and is meant to reference the idea that the entrance and exit to the camp is the ruined doorway pictured on the BG.  Essentially "the strange tunnels" are the halls of the ruins that the cats must traverse.  I don't think "winding" quite gives the same vibe, is there a different way we could imply the intention?

Reply:
oh gotcha, i missed that! hrm... is there a reason we can't add "angular" back into that description?

Reply:
I spose not, I removed it for shortening purposes but one additional word isn't bad lol